### ISSUE TITLE
Consider A11y in the editor | OS X Accessibility API

### ISSUE THREAD

**Issue by kleinerpirat** [ankitects/anki#2347]
A forum user notified me the mac app PopClip - a program that adds a popover menu for selected text - isn't working in Anki. This is because we're currently not following Apple's accessibility protocols / reporting our UI content to the accessibility API.

Here's a discussion on that in the PopClip forum: https://forum.popclip.app/t/popclip-not-working-in-anki/563/4

I consider this a relatively low priority while we're redesigning the UI, but if someone finds the time to evaluate the effort required to make Anki's webviews more accessible, please chime in!

---
Heads-up @hgiesel :v:

**Comment by dae** [comment]
I suspect this is something that needs to be solved by Qt rather than us, but if there's some easy solution I'm not aware of, I'd be happy to be proven wrong.

**Comment by samitopal** [comment]
I think this may have regressed in Anki 26.05.

PopClip works for me in Anki 25.09.4 (d52ca669), Python 3.13.5, Qt 6.9.1, Chromium 122. With the same deck/card templates, PopClip no longer appears in Anki 26.05 (e64c6b1a), Python 3.13.11, Qt 6.11.0, Chromium 140.

What still works in 26.05:
- Text selection is visibly highlighted in the reviewer.
- Cmd+C copies the selected text correctly.
- PopClip is enabled, and Anki is not excluded.
- PopClip works in other apps.
- PopClip does not appear to run its background-copy fallback in Anki; clipboard history shows no copy attempt.
- The same PopClip setup works in Anki 25.09.4.

So it looks like selected text is still available to Anki and the clipboard, but is no longer exposed in a way PopClip can detect through macOS accessibility/selection APIs. This may be related to the 26.05 Qt 6.11 / Chromium 140 / packaging change.

Happy to test a workaround or nightly build if helpful.

**Comment by abdnh** [comment]
Yes, it's probably a Qt issue, in which case there's probably not much we can do except wait for an upstream fix.
